product-service/serverless.ts

```diff
@@ -72,20 +94,44 @@ const serverlessConfiguration: AWS = {
         DeletionPolicy: 'Retain',
         Properties: {
           TableName: '${self:provider.environment.STOCKS_TABLE}',
-          AttributeDefinitions: [{
-            AttributeName: 'product_id',
-            AttributeType: 'S',
-          }],
-          KeySchema: [{
-            AttributeName: 'product_id',
-            KeyType: 'HASH',
-          }],
+          AttributeDefinitions: [
+            {
+              AttributeName: 'product_id',
+              AttributeType: 'S',
+            },
+          ],
+          KeySchema: [
+            {
+              AttributeName: 'product_id',
+              KeyType: 'HASH',
+            },
+          ],
           ProvisionedThroughput: {
             ReadCapacityUnits: 1,
             WriteCapacityUnits: 1,
           },
         },
       },
+      CatalogItemsQueue: {
+        Type: 'AWS::SQS::Queue',
+        Properties: {
+          QueueName: '${self:provider.environment.CATALOG_QUEUE_NAME}',
+        },
+      },
+      CreateProductTopic: {
+        Type: 'AWS::SNS::Topic',
+        Properties: {
+          TopicName: '${self:provider.environment.TOPIC_NAME}',
+        },
+      },
+      CreateProductTopicSubscription: {
```

Comment:
Could you please elaborate whether SNS filter policies are in the list of completed tasks?

_+15 (All languages) - set a Filter Policy for SNS createProductTopic in serverless.yml and create an additional email subscription to distribute messages to different emails depending on the filter for any product attribute_

Reply:
good catch, forgot to made the push but it's fixed now (: thanks